src/snowflake/cli/plugins/nativeapp/run_processor.py

```diff
@@ -145,15 +145,14 @@ def _execute_sql_script(self, sql_script_path):
         """
         with open(sql_script_path) as f:
             sql_script = f.read()
-            try:
-                if self.application_warehouse:
-                    self._execute_query(f"use warehouse {self.application_warehouse}")
```

Comment:
`_execute_sql_script` is only called from the context of creating/updating an app, where the warehouse has already been activated. Hence removed this additional query from here. But if folks feel strongly about this, it can be added back.

Reply:
Make sure to change the docstring to reflect this change. I agree it should not be here because it might be for the application _or_ for the application package.